src/main/java/teammates/ui/pagedata/InstructorFeedbackEditPageData.java

```diff
@@ -55,6 +58,10 @@ public void init(FeedbackSessionAttributes feedbackSession, List<FeedbackQuestio
                                       instructor.courseId, question, i + 1);
         }
 
+        this.courseDetails = courseDetails;
+        // numOfInstructors can be different from instructorList.size()
```

Comment:
Actually, by `instructorList` here, I meant the function's parameter, which is the variable currently in scope. `instructorList` may not have all the instructors (only ones with privileges are present). But `numOfInstructors` has the total number of instructors. Please look at [this line](https://github.com/TEAMMATES/teammates/pull/7509/files/d5b59c83a91317befd9cfeea12d68a63f9998b6e#diff-590458d6aeb0be2612217f008cc3818cR79). `instructorsWhoCanSubmit` is passed to this method's `instructorList`. So it may not have the total number of instructors.